ftw/keywordwidget/profiles/default/metadata.xml

```diff
@@ -1,4 +1,3 @@
 <?xml version="1.0"?>
 <metadata>
-    <version>1000</version>
```

Comment:
@mbaechtold since you have no upgrade step for `default` but you register it as upgrade step directory the initial version changes from `1000`  to something like `1000000000`. I'm not sure what the effect in the `@@manage-upgrades`-view is. Please verify that there are no warnings there.